set.mm

```diff
@@ -621195,26 +621286,25 @@ with respect to a linear (1-dimensional) relation.  (Contributed by BJ
   ${
     $d W a b l x y $.  $d K a b l x y $.  $d B a b x y $.
     0prjspn.w $e |- W = ( K freeLMod ( 0 ... 0 ) ) $.
-    0prjspn.0 $e |- .0. = ( 0g ` W ) $.
-    0prjspn.b $e |- B = ( ( Base ` W ) \ { .0. } ) $.
+    0prjspn.b $e |- B = ( ( Base ` W ) \ { ( 0g ` W ) } ) $.
```

Comment:
Just a remark: I have experimented from time to time with "inlining local definitions" (like `.0.` here). The statement looks better, but the usability will sometimes lessen a bit. There are tradeoffs.